recipes/sos/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "sos" %}
+{% set version = "0.17.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 470b891468c988885a3eada407cd85644c3ddf15a3b9d1a0993bffab755fbdee
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - sos=sos.__main__:main
+    - sos-runner=sos.__main__:sosrunner
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - fasteners
```

Comment:
can you remove the pieces you don't need from host

odds are all you need is python and pip

Reply:
I agree. These are runtime requirement, not host requirement.

Reply:
I tried to push the following patch
```diff
diff --git a/recipes/sos/meta.yaml b/recipes/sos/meta.yaml
index 6b61141a6..46e39dd5e 100644
--- a/recipes/sos/meta.yaml
+++ b/recipes/sos/meta.yaml
@@ -19,21 +19,8 @@ build:

 requirements:
   host:
-    - fasteners
-    - jinja2
-    - nbformat
-    - networkx
-    - pexpect
     - pip
-    - psutil
-    - pydot
-    - pydotplus
-    - pygments
     - python >=3.6
-    - pyyaml
-    - pyzmq
-    - setuptools
-    - tqdm
   run:
     - fasteners
     - jinja2
```

but got 
```
remote: Permission to conda-forge/staged-recipes.git denied to BoPeng.
fatal: unable to access 'https://github.com/conda-forge/staged-recipes.git/': The requested URL returned error: 403
```